apps/web/src/routes/_content/resources/newsletters/index.tsx

```diff
@@ -16,49 +19,134 @@ export const Route = createFileRoute('/_content/resources/newsletters/')({
 });
 
 function Newsletter() {
-  const [searchTerm, setSearchTerm] = useState('');
   const { t, i18n } = useTranslation();
+  const isMobile = useSmaller('md');
+
   const { data: newsletters, isFetched } = trpc.content.getNewsletters.useQuery(
-    {
-      language: i18n.language,
-    },
-    {
-      staleTime: 300_000, // 5 minutes
-    },
+    {},
+    { staleTime: 300_000 },
   );
 
-  const filteredNewsletters =
-    newsletters?.filter((newsletter) =>
-      newsletter.title?.toLowerCase().includes(searchTerm.toLowerCase()),
+  const localNewsletters =
+    newsletters?.filter(
+      (newsletter) => newsletter.language === i18n.language,
     ) ?? [];
+
+  const englishNewsletters =
+    newsletters?.filter((newsletter) => newsletter.language === 'en') ?? [];
+
+  const [showLocalOnly, setShowLocalOnly] = useState(false);
```

Comment:
should be true by default
![image](https://github.com/user-attachments/assets/3d6c6a9b-0c20-4709-9004-66b84cb2d12d)